### DISCUSSION TITLE
scalafx.org down?

### DISCUSSION THREAD

**Discussion by epic-64** [scalafx/scalafx#412] [Q&A] [answered]
Hey, I wasn't sure where else to put this and didn't find any piece of information about it.

Since a few days ago I can no longer access https://scalafx.org

In Chrome I get this error:
```
This site can’t be reached
Check if there is a typo in www.scalafx.org.
DNS_PROBE_FINISHED_NXDOMAIN
```

Is the website down for good? If not, are there plans to bring it back up?

**Comment by jpsacha** [comment]
I do not know what is going on with https://scalafx.org/, maybe the license for the domain run out, but that would probably result in some error message. If that persists I can remove forwarding to the custom domain and use `https://scalafx.github.io`

**Reply by epic-64** [reply]
Hmm yeah it's still unreachable due some DNS problem (I get the same error as above in Chrome and Firefox). Same for the github address since that forwards to the other one.

Quite demotivating as someone who is just getting started with ScalaFX (although I'm aware I could look up most things via JavaFX)

**Reply by jpsacha** [reply]
I am not in control of the scalafx.org domain. It was setup by @steveonjava . I will try to contact Steve and see if we can do something to bring the domain back.
For now I am removing forwarding to `scalafx.org` and using the default location for home site: [scalafx.github.io](https://scalafx.github.io). I will try to locate and correct the links across ScalaFX projects, though I will be traveling for the next couple of days, so there may be delays. If you see some places still with with `scalafx.org` please submit a PR with corrections

**Comment by steveonjava** [comment]
Sorry, the DNS record was at a hosting provider I stopped using, which 
broke the integration.

The domain is fine, so I switched where the DNS record was being stored 
and tried going through the verification steps in GitHub to reconnect 
it. It will either magically start working in the next 24h, or I will 
try troubleshooting their instructions again.

Cheers,
--Steve

**Reply by jpsacha** [reply]
@steveonjava I got request fro GitHub to verify scalafx.org domain that expires in 30 hours, but looks that only you can do that as you in control of it. It is a bit late request as I was traveling and had no way to login to GitHub. Here is a link, hope you can access:
[Manage verified domains](https://github.com/organizations/scalafx/settings/pages)
If you cannot get access, send me an email I will send you the instructions with hash codes (I do not want to place them in the open here). Thanks

**Comment by jpsacha** [comment, marked as answer]
@steveonjava thanks Steve. The https://scalafx.org/ is back online. I restored forwarding to https://scalafx.org/

**Comment by steveonjava** [comment]
I added the TXT record back in and verified the domain. Crisis averted. 😅

I think when I did the initial verification I was being tidy and deleted 
the TXT record after it verified. I guess they don't like that...
